airlock/views.py

```diff
@@ -2,15 +2,20 @@
 from django.shortcuts import redirect
 from django.template.response import TemplateResponse
 
-from airlock.workspace_api import PathItem
+from airlock.workspace_api import Workspace
 
 
 def index(request):
     return TemplateResponse(request, "index.html")
 
 
-def file_browser(request, path: str = ""):
-    path_item = PathItem.from_relative_path(path)
+def workspace_view(request, workspace_name: str, path: str = ""):
+    workspace = Workspace(workspace_name)
+    # TODO workspace authorization
+    if not workspace.exists():
+        raise Http404()
```

Comment:
Coverage is complaining about this line.

Reply:
Ah, thanks